client/client.go

```diff
@@ -31,6 +31,16 @@ func (c *Client) Query(ctx context.Context, method string, prefix types.Endpoint
 	return c.QueryStruct(queryCtx, method, prefix, path, in, &out)
 }
 
+// QueryRaw is a helper for initiating a request on any endpoints defined external to microcluster. This function should be used for all client
```

Comment:
Maybe remove `This function should be used for all client methods defined externally from microcluster` here since it's more of a special case

Reply:
Thanks for bringing this up.
Had the same thought as you but left it in to clearly indicate that you should not use the internal `QueryStruct/QueryStructRaw` methods.

But probably at some time we will anyway revoke access to those so we can already start thinking this way. Removed :)